src/lib/config/defaultConfig.js

```diff
@@ -241,6 +241,7 @@ export const RECORDS_CONFIG = {
       fields: {},
     },
     frontsiteMaxLinks: 5, // maximum number of links to show on details page
+    defaultLanguages: ['ENG'],
```

Comment:
I see that language is used also in the series... I would go for a more global config, as for the default currency, WDYT?

Reply:
I agree, it makes sense to add it also to the series. Thanks!